scipy/optimize/tests/test_linprog.py

```diff
@@ -1364,22 +1393,22 @@ def setup_method(self):
         self.options = {}
 
     def test_bug_5400(self):
-        with pytest.raises(ValueError):
-            super(TestLinprogSimplexDefault, self).test_bug_5400()
+#        with pytest.raises(ValueError):
+        super(TestLinprogSimplexDefault, self).test_bug_5400()
```

Comment:
I think we were checking for `ValueError` because the original behavior was erroneous success; these are now true successes.

Reply:
Here and below, rather than comment out the ``with pytest....`` I'd prefer to run these without scaling, just to check the errors are still raised.

Reply:
Yes. These occur when simplex "explodes" during a pivot operation and, while still feasible, is not optimal.

Reply:
Great. Well nice that it doesn't happen sometimes with autoscale on.